101-backup-create-disk-enable-protection/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create Disk & enable protection via Backup Vault",
+  "description": "Template that creates disk and enable protection via Backup Vault",
```

Comment:
Template that creates a disk and enables protection via a Backup Vault

Reply:
Addressed this.